crates/harp/src/string.rs

```diff
@@ -1,48 +0,0 @@
-//
-// string.rs
-//
-// Copyright (C) 2022 Posit Software, PBC. All rights reserved.
-//
-//
-
-use libr::ParseStatus;
-use libr::R_NaString;
-use libr::R_NilValue;
-use libr::R_ParseVector;
-use libr::Rf_xlength;
-use libr::EXPRSXP;
-use libr::SEXP;
-use libr::STRSXP;
-use libr::VECTOR_ELT;
-
-use crate::object::RObject;
-use crate::protect::RProtect;
-use crate::r_string;
-use crate::utils::r_typeof;
-
-// Given a quoted R string, decode it to get the string value.
-pub unsafe fn r_string_decode(code: &str) -> Option<String> {
```

Comment:
Not needed, this is `harp::parse_expr()` with an extra check for STRSXP at the end.